webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleToolbar.vue

```diff
@@ -23,49 +27,27 @@
         </option>
       </select>
     </v-scale-transition>
+    <v-scale-transition >
+      <v-btn
+          v-show="isMobile && showMobileFilter || !isMobile"
+          class="btn px-2 btn-primary"
+          outlined
+          @click="openPeopleAdvancedFilterDrawer()"
+      >
```

Comment:
Use Same formatting rules as other components: indentation = 2 spaces + > not in a single line

```suggestion
      <v-btn
        v-show="isMobile && showMobileFilter || !isMobile"
        class="btn px-2 btn-primary"
        outlined
        @click="openPeopleAdvancedFilterDrawer()">
```

Reply:
fixed